starter-code/main.css

```diff
@@ -0,0 +1,29 @@
+img {
+  width: 100%;
+}
+.navbar {
```

Comment:
Generally, it's better to create your own classes to manipulate. That way, you avoid unintended behavior that would arise from messing up with the bootstrap (or any other library) implementation.